modules/components/src/DataTable/Table/EnhancedReactTable.js

```diff
@@ -1,6 +1,6 @@
 import ReactTable from 'react-table';
 import { compose } from 'recompose';
-import checkboxHOC from 'react-table/lib/hoc/selectTable';
+import checkboxHOC from './checkboxHOC';
```

Comment:
can we try something like this? if it works, we can avoid having to maintain a fork of third-party libs :)

```
import checkboxHOC from 'react-table/lib/hoc/selectTable';

const defaultSelectInputComponent = props => {
  <input
    type={props.selectType || 'checkbox'}
    checked={props.checked}
    aria-label={`${props.checked ? 'Deselect' : 'Select'} ${props.id ? 'this row' : 'all rows'}`}
    onClick={e => {
      const { shiftKey } = e;
      e.stopPropagation();
      props.onClick(props.id, shiftKey, props.row);
    }}
    onChange={() => {}}
  />;
}

const withDefaultSelectInputComponent = Component => {
  return props => (
    <Component SelectInputComponent={props.SelectInputComponent || defaultSelectInputComponent} />
  )
}

const enhance = compose(
  checkboxHOC,
  withDefaultSelectInputComponent,
);
```

Reply:
This looks a lot better! Let me give it a try, thanks :)